### PR TITLE
fix flatfile reading in get_manta_sv

### PR DESCRIPTION
**Pull request by rdmorin**
# Pull Request Checklists

**Important:** When opening a pull request, keep only the applicable checklist and delete all other sections.

### Required

- [ ] I tested the new code for my use case (please provide a reproducible example of how you tested the new functionality)

- [ ] I ensured all dplyr functions that commonly conflict with other packages are fully qualified. 

This can be checked and addressed by running `check_functions.pl` and responding to the prompts. Test your code _after_ you do this.

- [ ] I generated the documentation and checked for errors relating to the new function (e.g. `devtools::document()`) and added `NAMESPACE` and all other modified files in the root directory and under `man`. 

### Optional but preferred with PRs

- [ ] I updated and/or successfully knitted a vignette that relies on the modified code (which ones?)

### Required

- [ ] I documented my function using [ROxygen style](https://jozef.io/r102-addin-roxytags/#:~:text=Inserting%20a%20skeleton%20%2D%20Do%20this,Shift%2BAlt%2BR%20).)

- [ ] All parameters for the function are described in the documentation and the function has a decriptive title. 

Example:
```
#' Use GISTIC2.0 scores output to reproduce maftools::chromoplot with more flexibility
#'
#' @param scores output file scores.gistic from the run of GISTIC2.0
#' @param genes_to_label optional. Provide a data frame of genes to label (if mutated). The first 3 columns must contain chromosome, start, and end coordinates. Another required column must contain gene names and be named `gene`. (truncated for example)
#' @param cutoff optional. Used to determine which regions to color as aberrant. Must be float in the range [0-1]. (truncated for example)
```

- [ ] My function uses a library that isn't already a dependency of GAMBLR and I made the package aware of this dependency using the function documentation `import` statment. 

Example:
```
#' @return nothing
#' @export
#' @import tidyverse ggrepel
```

## Checklist for changes to existing code

- [ ] I added/removed arguments to a function and updated documentation for all changed/new arguments

- [ ] I tested the new code for compatability with existing functionality in the Master branch (please provide a reprex of how you tested the original functionality)

### CODE REVIEW

**Review by Kdreval — APPROVED** [review]
Thanks!

File: R/annotation.R [hunk 1]
```diff
@@ -219,15 +219,15 @@ annotate_driver_ssm = function(maf_df,
...
   }else{
     #this all assumes the user has left the noncoding variants in their MAF
-    kept_ssm = maf_df %>% 
+    kept_ssm = maf_df %>%
       dplyr::filter(Variant_Classification %in% coding_vc) %>%
       dplyr::filter(Hugo_Symbol %in% driver_genes)
 
     for(gene in names(include_noncoding)){
       message(paste("adding", unname(include_noncoding[gene]), "for", gene))
-      nc_ssm = maf_df %>%
```
Comment (mattssca) [inline R/annotation.R:228]: Are all of these edits related to the feature you told us about last week, where Rstudio automatically cleans up white stray spaces etc? Handy!

File: R/annotation.R [hunk 2]
```diff
@@ -266,20 +266,20 @@ annotate_driver_ssm = function(maf_df,
...
-                       partner_bed, 
-                       with_chr_prefix = FALSE, 
-                       collapse_redundant = FALSE, 
+annotate_sv = function(sv_data,
+                       partner_bed,
+                       with_chr_prefix = FALSE,
+                       collapse_redundant = FALSE,
                        return_as = "bedpe_entrez",
-                       blacklist = c(60565248, 30303126, 187728894, 101357565, 101359747, 161734970, 69400840, 65217851, 187728889, 188305164),
+                       blacklist = c(60565248, 30303126, 187728894, 101357565, 101359747, 161734970, 69400840, 65217851, 187728889, 187728888,187728892, 187728893,188305164),
```
Comment (mattssca) [inline R/annotation.R:274]: Should this blacklist parameter I implemented be pointing to a bed file instead? Or is it enough, for now, to just state such regions like this?

**Review by mattssca — APPROVED** [review]
Thanks for the updates, Ryan. Do you want to regenerate the documentation to have it reflect the added (and updated) parameters or this can wait until the remaining issues are resolved?

File: R/annotation.R (hunk 2 above)
Comment (rdmorin) [reply R/annotation.R:274]: We should definitely move this information to a bundled file

**Pull request merged**